conans/client/migrations.py

```diff
@@ -44,7 +44,7 @@ def _make_migrations(self, old_version):
 arch: [x86, x86_64, ppc64le, armv6, armv7, armv7hf, armv8]
 compiler:
     gcc:
-        version: ["4.4", "4.5", "4.6", "4.7", "4.8", "4.9", "5.1", "5.2", "5.3", "5.4", "6.1", "6.2"]
```

Comment:
The new setting doesn't have to be included here. This is a migration, will use the ``old_settings`` to know if the user changed something from the previous versions defaults.